src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNodeTests.cs

```diff
@@ -679,7 +679,9 @@ public void TestFindTokenInLargeList()
                 missingArgument, missingComma,
                 argument);
 
-            var argumentList = SyntaxFactory.ArgumentList(SyntaxFactory.SeparatedList<ArgumentSyntax>(SyntaxFactory.NodeOrTokenList(nodesAndTokens)));
+            var argumentList = SyntaxFactory.ArgumentList(SyntaxFactory.SeparatedList<ArgumentSyntax>(collectionExpression
```

Comment:
>SyntaxFactory.SeparatedList<ArgumentSyntax>( [](http://example.com/codeflow?start=58&length=44)

nit: Do we need the `SyntaxFactory.SeparatedList<ArgumentSyntax>(` part in the collection expression case?

Reply:
probably not.  but i was trying to test out the NodeOrTokenList bypass.